### PR TITLE
chore(curriculum): remove pyodide from shortest path algorithm project

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

https://github.com/freeCodeCamp/freeCodeCamp/blob/08f63a0b974c5ca17aa2b27b3ce5151fd0b2eea6/tools/client-plugins/browser-scripts/python-test-evaluator.ts#L122-L123

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
No pyodide found in this block

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6578b13757611e2825beb8a5.md [hunk 1]
```diff
@@ -26,6 +27,7 @@ You should have an `age` key with the value `2` just after `'species': 'guinea p
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
+    copper = __locals.get("copper")
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6578b13757611e2825beb8a5.md:30]:
```suggestion
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6578b13757611e2825beb8a5.md [hunk 2]
```diff
@@ -26,6 +27,7 @@ You should have an `age` key with the value `2` just after `'species': 'guinea p
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6578b13757611e2825beb8a5.md:29]:
```suggestion
({ test: () => assert(runPython(`
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6578b13757611e2825beb8a5.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ You should add a new key-value pair to your `copper` dictionary.
...
+    copper = __locals.get("copper")
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6578b13757611e2825beb8a5.md:20]:
```suggestion
({ test: () => assert(runPython(`
```

**Review by Dario-DC — CHANGES_REQUESTED** [review]
Something went wrong when fixing the conflicts

**Review by Dario-DC — APPROVED** [review]

**Pull request merged**